src/ProjectService.Business/Commands/Image/RemoveImageCommand.cs

```diff
@@ -0,0 +1,126 @@
+        private readonly IRemoveImageValidator _validator;
+
+        private bool RemoveImage(List<Guid> ids, List<string> errors)
+        {
+            if (ids == null || ids.Count == 0)
+            {
+                return false;
+            }
+
+            string logMessage = "Errors while removing images ids {ids}. Errors: {Errors}";
+
+            try
+            {
+                Response<IOperationResult<bool>> brokerResponse = _rcImages.GetResponse<IOperationResult<bool>>(
+                   IRemoveImagesRequest.CreateObj(ids, ImageSource.Project)).Result;
+
+                if (brokerResponse.Message.IsSuccess && brokerResponse.Message.Body)
+                {
+                    return true;
+                }
+
+                _logger.LogWarning(
+                    logMessage,
+                    string.Join('\n', ids),
+                    string.Join('\n', brokerResponse.Message.Errors));
+            }
+            catch (Exception exc)
+            {
+                _logger.LogError(exc, logMessage);
+            }
+
+            errors.Add("Can not remove images. Please try again later.");
+
+            return false;
+        }
+
+        public RemoveImageCommand(
+            IImageRepository repository,
+            IRequestClient<IRemoveImagesRequest> rcImages,
+            ILogger<RemoveImageCommand> logger,
+            IAccessValidator accessValidator,
+            IHttpContextAccessor httpContextAccessor,
+            IRemoveImageValidator validator)
+        {
+            _repository = repository;
+            _rcImages = rcImages;
+            _logger = logger;
+            _accessValidator = accessValidator;
+            _httpContextAccessor = httpContextAccessor;
+            _validator = validator;
+        }
+
+        public OperationResultResponse<bool> Execute(List<RemoveImageRequest> request)
+        {
+            OperationResultResponse<bool> response = new();
+
+            if (!_accessValidator.HasRights(Rights.AddEditRemoveProjects))
+            {
+                _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.Forbidden;
+                response.Status = OperationResultStatusType.Failed;
+                response.Errors.Add("Not enough rights.");
+
+                return response;
+            }
+
+            List<string> errors = new();
+
+            if (!_validator.ValidateCustom(request, out errors))
+            {
+                _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                response.Status = OperationResultStatusType.Failed;
+                response.Errors.AddRange(errors);
+
+                return response;
+            }
+
+            List<Guid> imagesIds = request
+                .Select(x => x.ImageId)
+                .ToList();
+
+            bool result = RemoveImage(imagesIds, response.Errors);
+
+            if (response.Errors.Any() && !result)
```

Comment:
just !result
if result if false - there always be an errors

Reply:
done